pkg/deployments/package.json

```
@@ -25,6 +25,7 @@
     "check-artifacts": "hardhat check-artifacts",
     "check-deployments": "hardhat check-deployments --network mainnet && hardhat check-deployments --network polygon && hardhat check-deployments --network arbitrum && hardhat check-deployments --network optimism && hardhat check-deployments --network goerli",
     "check-action-ids": "hardhat check-action-ids --network mainnet && hardhat check-action-ids --network polygon && hardhat check-action-ids --network arbitrum && hardhat check-action-ids --network optimism",
+    "extract-artifacts": "hardhat extract-artifacts --id",
```

```suggestion
    "extract-artifacts": "hardhat extract-artifacts",
```

I don't like how we're making `id` into a positional parameter here. If we want it to be positional then we should modify the task but I would leave it as it is.